tools/extra/mmlink/remote_dbg/streaming/server.c

```diff
@@ -375,6 +375,8 @@ const char *get_parameter(char *cmd, SERVER_CONN *server_conn) {
 const char *set_parameter(char *cmd, SERVER_CONN *server_conn, CLIENT_CONN *client_conn) {
     const char *param_name = strstr(cmd, SET_PARAM_CMD) + SET_PARAM_CMD_LEN;

```

Comment:
If strstr returns NULL, the check on 378 won't fire, because param_name will be at least SET_PARAM_CMD_LEN.